frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containersList.component.tsx

```diff
@@ -70,52 +84,99 @@ export const ContainersList = ({
 		}
 	};
 
-	return useMemo(() => (
+	return (
 		<Container>
 			<DashboardListCollapse
 				title={<>{title} {!isListPending && `(${containers.length})`}</>}
 				tooltipTitles={titleTooltips}
 				isLoading={areStatsPending}
+				sideElement={(
+					<CollapseSideElementGroup>
```

Comment:
Question: Is it necessary to create this element? Can it be a FixedOrGrowContainer?